src/components/TopNav.js

```diff
@@ -66,4 +67,12 @@ function TopNav(props) {
     </ul>);
 }
 
-export default TopNav;
+
```

Comment:
Good job here. This is where the props will be passed into the components now, so you can remove the props passed into them from the original parent component. It is now done in the TopNavContainer.